### DISCUSSION TITLE
Discontinue folder expansion in ListView

### DISCUSSION THREAD

**Discussion by jeremypw** [elementary/files#1692] [Ideas]
We have folder expansion in both ListView and ColumnView.  Nautilus ListView defaults to having folder expansion turned off, although it can be turned on through a preference.

Personally I find using folder expansion in ListView unwieldy and hardly ever use it.  ColumnView is much better for drilling into multiple depths of folders.

It is unclear what advantages using ListView for folder expansion has over ColumnView.  Discontinuing it would simplify the model and reduce wasted space on the left.

**Comment by hanaral** [comment]
As someone who doesn't ever use listview, my opinion isn't as valid as some one who does. However, if this means that all three types can do specifically their own thing, better, then I'm all for it.
